lib/sycamore/sycamore/query/execution/sycamore_operator.py

```diff
@@ -299,34 +300,38 @@ def script(self, input_var: Optional[str] = None, output_var: Optional[str] = No
         assert isinstance(self.logical_node, BasicFilter)
         assert self.logical_node.dependencies is not None and len(self.logical_node.dependencies) == 1
         imports: list[str] = []
+
+        input_str = input_var or get_var_name(self.logical_node.dependencies[0])
+        output_str = output_var or get_var_name(self.logical_node)
         if self.logical_node.range_filter:
             field = self.logical_node.field
             start = self.logical_node.start
             assert start is None or isinstance(start, str)
             end = self.logical_node.end
             assert end is None or isinstance(end, str)
             date = self.logical_node.date
-
-            script = (
-                f"{output_var or get_var_name(self.logical_node)} = "
-                f"{input_var or get_var_name(self.logical_node.dependencies[0])}.filter(\n"
-                "f=RangeFilter("
-                f"field='{field}',\n"
-                f"start='{start}',\n"
-                f"end='{end}',\n"
-                f"date='{date}),'\n"
-                f"**{self.get_node_args()})"
-            )
+            script = f"""
+{output_str} = {input_str}.filter(
+    f=RangeFilter(
+        field='{field}',
```

Comment:
Note that start and end values might be ints (for example) so wrapping them in quotes is probably not what you want to do in those cases. We should probably have a utility function to render parameters being passed into the generated code that handles things like that.

Reply:
Realized we're going to need input/output schemas (or types) for these operators - currently there isn't a clear way to easily determine what the type of input data is. We had plans for them but seems like now is a good time to add them